workspace/viewer/PdfJsViewer.ts

```diff
@@ -505,19 +513,6 @@ export class PdfJsViewer {
     }
   }
 
-  pageUp = () => {
```

Comment:
We need these functions in production, please get them back and make them working.

Reply:
Done.